packages/kit/src/views/Setting/pages/Tab/SubSettingsPage.tsx

```
@@ -0,0 +1,67 @@
+  name,
+  settingsConfig,
+}: {
+  name: ISettingName;
+  settingsConfig: ISettingsConfig;
+}) {
+  const isTabNavigator = useIsTabNavigator();
+  const configList = useMemo(() => {
+    return settingsConfig
+      ? settingsConfig
+          ?.find((item) => item?.title === name)
+          ?.configs.filter((item) => item && item.length)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)